examples/jaeger-tracing/front-envoy-jaeger.json

```diff
@@ -20,7 +20,10 @@
                   "routes": [
                     {
                       "prefix": "/",
-                      "cluster": "service1"
+                      "cluster": "service1",
+                      "decorator": {
```

Comment:
`decorator` seems to be pretty generic name, should it be tracing_decorator. And also we need doc update on this.

Reply:
Using the more general `decorator` term was @mattklein123 's idea: https://github.com/lyft/envoy-api/pull/171#discussion_r138397108

Will sort out docs soon.

Reply:
fair enough